cachelib/allocator/CacheAllocator-inl.h

```diff
@@ -2940,7 +2916,7 @@ bool CacheAllocator<CacheTrait>::markMovingForSlabRelease(
     // Since this callback is executed, the item is not yet freed
     itemFreed = false;
     Item* item = static_cast<Item*>(memory);
-    if (item->markMoving()) {
+    if (item->markExclusive()) {
```

Comment:
This now returns false if the item was already marked as exclusive. This causes issue in slab release.

Reply:
In case it returns false, we will just spin in the while loop until the item is either freed or we succeed in marking it as exclusive.

Reply:
yeah you're right. This will just repeat in the loop. I noticed one more thing L2956 that I think we should remove the assert there.